app/controllers/corporate_bodies_controller.rb

```diff
@@ -1,106 +1,129 @@
+# frozen_string_literal: true
+
 class CorporateBodiesController < ApplicationController
-  before_action :set_corporate_body, only: %i[ show edit update destroy corp_toc]
+  before_action do |c| c.require_editor('edit_people') end
```

Comment:
Perhaps we may want to add another bit to edit corporates, but for now I've used same check as for persons.

Reply:
Yeah, they're the same bit.